vscode/package.json

```diff
@@ -132,7 +132,9 @@
   "scripts": {
     "tsc:check": "node ../node_modules/typescript/bin/tsc -p ./tsconfig.json",
     "build": "npm run tsc:check && node build.mjs",
-    "run:web": "npx @vscode/test-web --extensionDevelopmentPath . ../samples"
+    "run:web": "npx --no-install @vscode/test-web --extensionDevelopmentPath . ../samples",
```

Comment:
Interesting note which may bite us in the future, but apparently 'npx' is deprecated and an alias for 'npm exec' now, and the `--no-install` option is also deprecated (replaced with `--no`)

See https://docs.npmjs.com/cli/v9/commands/npm-exec?v=true#compatibility-with-older-npx-versions